libstuff/libstuff.cpp

```diff
@@ -2563,11 +2563,7 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     string sqlToLog = sql;
     if ((int64_t)elapsed > warnThreshold) {
         // This code removing authTokens is a quick fix and should be removed once https://github.com/Expensify/Expensify/issues/144185 is done.
-        string match;
-        const bool hasAuthToken = SREMatch(".*(\"authToken\"\\:\"[0-9A-F]{400,1024}\").*", sql, match);
-        if (hasAuthToken) {
-            sqlToLog = SReplace(sql, match, "<REDACTED_AUTHTOKEN>");
-        }
+        pcrecpp::RE("\"authToken\":\"[0-9A-F]{400,1024}\"").GlobalReplace("<REDACTED_AUTHTOKEN>", &sqlToLog);
```

Comment:
Can we leave the `"authToken:"` part of the log line unredacted, and use `<REDACTED>` as the string to redact since that's what we use in PHP for sensitive keys?

eg, `VALUES ( JSON('{"accountID":2543349,"authToken":<REDACTED>,"reportID":84035592...`